test/functional/dbcrash.py

```diff
@@ -99,14 +87,16 @@ def restart_node(self, node_index, expected_tip):
         # and make sure that recovery happens.
         raise AssertionError("Unable to successfully restart node %d in allotted time", node_index)
 
-    # Try submitting a block to the given node.
-    # Catch any exceptions that indicate the node has crashed.
-    # Returns true if the block was submitted successfully; false otherwise.
     def submit_block_catch_error(self, node_index, block):
+        """Try submitting a block to the given node.
+
+        Catch any exceptions that indicate the node has crashed.
+        Returns true if the block was submitted successfully; false otherwise."""
+
         try:
             self.nodes[node_index].submitblock(block)
             return True
-        except (httplib.CannotSendRequest, httplib.RemoteDisconnected) as e:
+        except (http.client.CannotSendRequest, http.client.RemoteDisconnected) as e:
```

Comment:
I think I've fixed this in my fixup commit b394a00f6c34e20e5ff555c8a07c6558e319a34a. Just catch different errors depending on which version of Python is being run.

I think later we should move this logic to a separate method named `send_rpc_may_fail()` or similar in TestNode to keep the test script code clean.